templates/taxi/driver_list.html

```diff
@@ -6,8 +6,14 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
```

Comment:
Code Efficiency: The code indentation is not consistent. For better readability and maintainability, it is recommended to use 2 whitespaces indentation in your `.html` files.